### PR TITLE
Banned user check for profile view

### PR DESCRIPTION
**Pull request by lyyder**
Adds a check that user is not banned when loading a user profile.

### CODE REVIEW

File: app/controllers/people_controller.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ class PersonDeleted < StandardError; end
 
   before_filter EnsureCanAccessPerson.new(
     :id, error_message_key: "layouts.notifications.you_are_not_authorized_to_view_this_content"), only: [:update, :destroy]
+  #before_filter :ensure_person_not_banned, :only => [:show, :update]
```
Comment (rap1ds) [inline app/controllers/people_controller.rb]: This seems to be commented out. By accident?

**Comment by rap1ds** [comment]
There seems to be already a quite handy way implemented for checking if the person is deleted. Probably, adding a new PersonBanned exception would've been the easiest way to implement this. Maybe you should consider doing it that way?

File: app/controllers/people_controller.rb (hunk 1 above)
Comment (lyyder) [inline app/controllers/people_controller.rb]: Oh, nice catch!

**Comment by lyyder** [comment]
@rap1ds Sure, I'll have a look.

**Comment by rap1ds** [comment]
LGTM

**Pull request merged**